### PR TITLE
Add common issues section to Readme.md

### PR DESCRIPTION
**Pull request by milan-tom**
I have added a common issues section to the README.md file. As per the request of @codecakes, this will help others facing similar common issues, making the setup process quicker.

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/52)

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -84,7 +84,7 @@ pip_repositories()
 # requirements.txt.
 pip_import(
    name = "my_deps",
-   requirements = "//:requirements.txt",
+   requirements = ":requirements.txt",
```
Comment (codecakes) [inline WORKSPACE:87]: pull changes to ur master and merge to this branch; this is nit;

**Review by codecakes — CHANGES_REQUESTED** [review]
@milan-tom make the changes and re request review

**Comment by codecakes** [comment]
@milan-tom you can include the changes right here instead of #56

**Comment by milan-tom** [comment]
> @milan-tom you can include the changes right here instead of #56

@codecakes do you want me to close #56?

**Pull request closed without merging**